com.lunarclient.Launcher.yml

```
@@ -0,0 +1,59 @@
+app-id: com.lunarclient.Launcher
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+branch: stable
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '21.08'
+command: lunarclient
+separate-locales: false
+rename-icon: lunarclient
+finish-args:
+  - --persist=.minecraft
+  - --persist=.lunarclient
+  - --share=network
+  - --device=dri
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+modules:
+  - name: lunarclient
+    buildsystem: simple
+    cleanup:
+      - squashfs-root
+    sources:
+      - type: extra-data
+        url: https://launcherupdates.lunarclientcdn.com/Lunar%20Client-2.9.3.AppImage
+        size: 89608553 
+        sha256: db0942f8e006da5bd4077c8fc69a4d861bcd0d9bf8831acd2a9c30f6f01f4ad2
+        filename: lunar-client-2.9.3.appimage
+        
+      - type: script
+        dest-filename: lunarclient
+        commands:
+          - exec zypak-wrapper /app/lunarclient/lunarclient --no-sandbox "$0"
+
+      - type: file
+        path: com.lunarclient.Launcher.appdata.xml
+
+      - type: file
+        path: com.lunarclient.Launcher.desktop
+      
+      - type: file
+        path: apply_extra.sh
+      
+      - type: file
+        path: icons/lunarclient.png
+
+    build-commands:
+      - install -D ./apply_extra.sh "${FLATPAK_DEST}/bin/apply_extra"
+      - install lunarclient "${FLATPAK_DEST}/bin/"
+      - install -Dm644 com.lunarclient.Launcher.desktop "${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop"
+      - install -Dm644 com.lunarclient.Launcher.appdata.xml "${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.appdata.xml"
+      - install  -Dm644 -t ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps install/lunarclient.png
```

```suggestion
      - install  -Dm644 -t ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps icons/lunarclient.png
```

Gosh, I messed up the original suggestion.